docker/.env

```
@@ -266,7 +272,12 @@ SMQ_VERIFICATION_URL_PREFIX=http://localhost/verify-email
 SMQ_VERIFICATION_EMAIL_TEMPLATE=verification-email.tmpl
 
 #### Users Client Config
-SMQ_USERS_URL=users:9002
+SMQ_USERS_URL=http://users:9002
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The SMQ_USERS_URL key is duplicated

(DuplicatedKey)